database/user.js

```diff
@@ -0,0 +1,36 @@
+const pgp = require( 'pg-promise' )()
+const db = pgp( process.env.DATABASE_URL )
+const bcrypt = require( 'bcrypt-nodejs' )
+
+const FIND_BY_EMAIL = `SELECT * FROM users WHERE email=$1`
+
+const FIND_BY_ID = `SELECT * FROM users WHERE id=$1`
```

Comment:
Looks like 2 of the queries use backtick-style strings, and the others use single-quote strings. Can you please update to use all the same kind of strings?